packaging/opensuse/snapd.spec

```diff
@@ -118,12 +142,15 @@ CXXFLAGS="$RPM_OPT_FLAGS -fPIC -Wl,-z,relro -Wl,-z,now"
 export CFLAGS
 export CXXFLAGS
 
-# NOTE: until snapd and snap-confine have the improved communication mechanism
-# we need to disable apparmor as snapd doesn't yet support the version of
-# apparmor kernel available in SUSE and Debian. The generated apparmor profiles
-# cannot be loaded into a vanilla kernel. As a temporary measure we just switch
-# it all off.
-%configure --disable-apparmor --libexecdir=%{_libexecdir}/snapd
+# N.B.: Prior to openSUSE Tumbleweed in May 2018, the AppArmor userspace in SUSE
+# did not support what we needed to be able to turn on basic integration.
+%configure \
+    %{!?with_apparmor:--disable-apparmor} \
+    --libexecdir=%{_libexecdir}/snapd \
+    --enable-nvidia-biarch \
+    %{?with_multilib:--with-32bit-libdir=%{_prefix}/lib} \
+    --with-snap-mount-dir=%{snap_mount_dir} \
+    --enable-merged-usr
```

Comment:
This is the problem, don't pass `--enable-merged-usr`.

Reply:
openSUSE uses `/run/media`, though...

Reply:
So maybe the test needs an update. Let me look into this.

Reply:
@Conan-Kudo  Looks like I can't push to your branch. Can you pick up this patch? https://github.com/bboozzoo/snapd/commit/48352a6eefff56c0f5bb77fb814ca852139630f7

Reply:
Done.